lib/modules/codemirror_module.dart

```diff
@@ -11,13 +11,7 @@ import '../editing/editor_codemirror.dart';
 class CodeMirrorModule extends Module {
   static String get version => codeMirrorFactory.version;
 
-  Future init() {
+  Future init() async {
```

Comment:
why async here?

Reply:
to not have to return an explicit `return Future.value()`.